models/EventModel.ts

```diff
@@ -59,6 +60,9 @@ export class EventModel extends BaseEntity {
   @OneToMany((type) => AttendanceModel, (attendance) => attendance.event, { cascade: true })
   attendances: AttendanceModel[];
 
+  @OneToMany((type) => ExpressCheckinModel, (expressCheckin) => expressCheckin.event, { cascade: true })
+  expressCheckins: AttendanceModel[];
```

Comment:
Does this add column need to be specified in a migration?

Reply:
Nope, this is purely a TypeORM construct. This field allows `EventModel` to join with `ExpressCheckinModel` and be accessed with `event.expressCheckins`, for ease of use. Internally in the database though, this relationship only needs to be one-way - i.e., `ExpressCheckins.event` references the `Events` table, but the `Events` table doesn't need to know about `ExpressCheckins`, unless it's specifically queried via. TypeORM.

More about this behavior here: https://www.tutorialspoint.com/typeorm/typeorm_relations.htm